src/items/constant-items.md

```diff
@@ -49,7 +49,26 @@ const BITS_N_STRINGS: BitsNStrings<'static> = BitsNStrings {
 ```
 
 r[items.const.final-value-immutable]
-The final value of a `const` item cannot contain references to anything mutable.
+The final value of a `const` item cannot contain a mutable reference.
```

Comment:
This is pretty much an entirely new clause, at least in terms of intent, so IMO it'd make sense to give it a new name, such as `items.const.no-mutable-ref`.

Reply:
Thanks, done.